### PR TITLE
Add docs talking about how to adding symbolic for unsupported ops

### PR DESCRIPTION
**Pull request by houseroad**
Restructured the doc from https://github.com/onnx/tutorials/blob/master/tutorials/PytorchAddExportSupport.md.

@dzhulgakov 

@ezyang don't want to ruin your vacation, so you can just ignore this. :-)

### CODE REVIEW

**Comment by apaszke** [comment]
I think that we'll be giving `Value`s to `symbolic`, not `Node`s once #3705 is merged

**Comment by houseroad** [comment]
Thanks, I will update the API specification accordingly.

**Comment by dzhulgakov** [comment]
Awesome, can we also update the list of supported ops? (maybe at some point we should make it autogenerated :) )

**Comment by houseroad** [comment]
Sure!

**Comment by houseroad** [comment]
I think probably later, almost all the ops will go through ATen, we can just simply grap the supported ops from symbolic.py. Life becomes easy. :-)

**Comment by houseroad** [comment]
@dzhulgakov Looks like @ezyang updated it here :-)
https://github.com/pytorch/pytorch/pull/3957

File: docs/source/onnx.rst [hunk 1]
```diff
@@ -161,8 +162,126 @@ list.  The operator set above is sufficient to export the following models:
 * VGG
 * `word_language_model <https://github.com/pytorch/examples/tree/master/word_language_model>`_
 
-The interface for specifying operator definitions is highly experimental
-and undocumented; adventurous users should note that the APIs will probably
+If the wanted operator is standardized in ONNX, it should be easy to add
+support for exporting such operator (adding a symbolic funtion for the operator).
+To confirm whether the operator is standardized or not, please check the `ONNX oeprator list <http://https://github.com/onnx/onnx/blob/master/docs/Operators.md>`_.
```
Comment (ezyang) [inline docs/source/onnx.rst]: s/oeprator/operator/

File: docs/source/onnx.rst [hunk 2]
```diff
@@ -161,8 +162,126 @@ list.  The operator set above is sufficient to export the following models:
 * VGG
 * `word_language_model <https://github.com/pytorch/examples/tree/master/word_language_model>`_
 
-The interface for specifying operator definitions is highly experimental
-and undocumented; adventurous users should note that the APIs will probably
+If the wanted operator is standardized in ONNX, it should be easy to add
```
Comment (ezyang) [inline docs/source/onnx.rst]: This section needs to more clearly state that adding operators is advanced usage; for example, some of the instructions below suggest that you look into generated source files, but those instructions only make sense for users which have done for-source builds.

**Comment by ezyang** [comment]
@houseroad  Don't forget about this PR!

**Comment by houseroad** [comment]
Thanks for reminding me. Was fighting with cmake. Done :-)

**Pull request merged**